index.js

```diff
@@ -1,5 +1,31 @@
 'use strict';
 var chalk = require('chalk');
+var isSupported = (function() {
+	if (process.platform !== 'win32') {
+		return true;
+	}
+
+	var env = process.env;
+
+	if (env.APPVEYOR) {
```

Comment:
Yes, or prove it works. You could run a AppVeyor build with `console.log('✔')` and take a screenshot.